alfresco-integration/shared/src/main/java/com/github/dynamicextensionsalfresco/models/M2ModelResourceListProvider.java

```diff
@@ -0,0 +1,67 @@
+package com.github.dynamicextensionsalfresco.models;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.alfresco.error.AlfrescoRuntimeException;
+import org.alfresco.repo.dictionary.M2Model;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ResourceLoaderAware;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.core.io.support.ResourcePatternResolver;
+import org.springframework.util.Assert;
+
+public class M2ModelResourceListProvider implements M2ModelListProvider, ResourceLoaderAware {
+
+    private Logger logger = LoggerFactory.getLogger(M2ModelResourceListProvider.class);
+
+    private final String modelLocationPattern = "osgibundle:/META-INF/alfresco/models/*.xml";
```

Comment:
Make this constant `private static final` to avoid duplication when new class instances are made.